cmd/frontend/db/repos.go

```diff
@@ -136,10 +157,12 @@ var getBySQLColumns = []string{
 }
 
 func (s *repos) getBySQL(ctx context.Context, querySuffix *sqlf.Query) ([]*types.Repo, error) {
-	return s.getReposBySQL(ctx, false, querySuffix)
+	return s.getReposBySQL(ctx, true, false, querySuffix)
 }
 
-func (s *repos) getReposBySQL(ctx context.Context, minimal bool, querySuffix *sqlf.Query) ([]*types.Repo, error) {
+// 🚨 SECURITY: It is the caller's responsibility to ensure the current authenticated user
+// is the site admin who could see all the repositories when enforcePermission=false.
```

Comment:
Hmm, I'm not sure I understand the described behavior. Is this what you meant?

```suggestion
// is the site admin who is authorized to see the repositories returned even when enforcePermission=false.
```